src/tests/ascii/CMakeLists.txt

```diff
@@ -1,9 +1,4 @@
-include_directories(${PROJECT_BINARY_DIR}/src)
```

Comment:
Thanks for removing it, this was a bad design that slipped through in our review of #23 and #32 (which just copied it).

Reply:
Yeah, this command should never be used.